src/test/java/com/microsoft/azure/proton/transport/ws/impl/WebSocketImplTest.java

```diff
@@ -37,20 +37,21 @@
 import static org.mockito.Mockito.when;
 
 public class WebSocketImplTest {
-    private int _allocatedWebSocketBufferSize = (4 * 1024) + (16 * WebSocketHeader.MED_HEADER_LENGTH_MASKED);
-    private String _hostName = "host_XXX";
-    private String _webSocketPath = "path1/path2";
-    private String _webSocketQuery = "";
-    private int _webSocketPort = 1234567890;
-    private String _webSocketProtocol = "subprotocol_name";
-    private Map<String, String> _additionalHeaders = new HashMap<String, String>();
 
-    private int _lengthOfUpgradeRequest = 284;
+    private int allocatedWebSocketBufferSize = (4 * 1024) + (16 * WebSocketHeader.MED_HEADER_LENGTH_MASKED);
```

Comment:
Could these all be `static final` or `final`? Or are they reassigned elsewhere

Reply:
Good that you have noticed it, these variables are just used for assertions, made them as constants, by using `private static final` access specifier.